backend/src/project/dto/project.input.ts

```diff
@@ -0,0 +1,17 @@
+// DTOs for Project APIs
+import { InputType, Field, ID } from '@nestjs/graphql';
+
+@InputType()
+export class UpsertProjectInput {
+  @Field()
+  project_name: string;
+
+  @Field()
+  path: string;
```

Comment:
We don't have to explore path to user, this path is absolute path of storage in backend server